api_tests/e2e/rfc003/btc_eth-erc20/test.js

```diff
@@ -187,7 +200,8 @@ describe("RFC003: Bitcoin for ERC20", () => {
         await alice.poll_comit_node_until(
             chai,
             alice_swap_href,
-            "AlphaFunded"
+            "AlphaFunded",
+            true
```

Comment:
❓What's this `true` doing here?

Reply:
@bonomat it's new, it is to say that you want to generate bitcoin blocks when polling. Look at the new definition of `poll_comit_node_until`.

Reply:
@LLFourn thinks we keep the ticker. I'll see what I can do.